utils/serverConfig.ts

```
@@ -0,0 +1,186 @@
+  return lines.join('\n');
+}
+
+export async function loadResolvedServerConfig(
+  db: ServerConfigReader,
+  serverId: string,
+): Promise<ResolvedServerConfig> {
+  const rows = await db.serverConfig.getAllServerConfig(serverId);
+  const values = new Map(rows.map((row) => [row.key, row.value]));
+
+  return {
+    pokemonSpawnRate: parseRate(values.get(SERVER_CONFIG_KEYS.POKEMON_SPAWN_RATE), DEFAULT_RATES[SERVER_CONFIG_KEYS.POKEMON_SPAWN_RATE]),
```

ESLint

[error] This line has a length of 137. Maximum allowed is 120.

(max-len)